File: 171._Excel_Sheet_Column_Number.cpp
class Solution {
public:
    int titleToNumber(string s) {
        int n=s.size();
        int col=0;
        for(int i=0;i<n-1;i++){
            col+=(s[i]-64)*pow(26,n-i-1);
        }
        int last=s[n-1]-64;
        cout<<col<<" "<<last;
        return col+last;
    }
};

File: 171._Excel_Sheet_Column_Number.py
class Solution:
    def titleToNumber(self, s):
        n=len(s)
        col=0
        for i in range(n-1):
            col=col+(ord(s[i])-64)*math.pow(26,n-1-i)
        last=ord(s[-1])-64
        return int(col+last)